internal/engine/upper_test.go

```diff
@@ -1516,180 +1517,6 @@ func TestPodEventContainerStatusWithoutImage(t *testing.T) {
 	assert.Nil(t, err)
 }
 
-func TestPodUnexpectedContainerStartsImageBuild(t *testing.T) {
```

Comment:
these tests were doing so much disabling/mocking of things that i just don't think they're useful anymore